.bazelversion

```diff
@@ -0,0 +1 @@
+2.0.0
```

Comment:
This is fairly old by now. Maybe 3.0.0?

Reply:
Updated to the latest 3.5.0. We'd need to update the version in upstream proxy-wasm-cpp-sdk as well.